assets/js/blocks/cart-checkout/checkout/style.scss

```
@@ -125,6 +125,15 @@
 		border-radius: 3px;
 		height: auto;
 		margin-left: auto;
+
+		.wc-block-components-button__text {
+			line-height: 24px;
+
+			svg {
```

Again, I'm a bit wary of using tag selectors. :slightly_smiling_face:  Could we use a child combinator here? This way, if some day we want to add an inner component which has other `svg` elements, they would not be affected by this rule.
```suggestion
			> svg {
```